miniti-client/src/app/ticket-list/ticket-list.component.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { TicketListComponent } from './ticket-list.component';
+import {TicketEntryComponent} from '../ticket-entry/ticket-entry.component';
+
+describe('TicketListComponent', () => {
+  let component: TicketListComponent;
+  let fixture: ComponentFixture<TicketListComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ TicketListComponent, TicketEntryComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(TicketListComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
I don't know if this would worth the effort but maybe a good test for this component is check it shows the right amount of entries